src/math/State.cpp

```diff
@@ -87,4 +87,31 @@ double State::rel_az(const Eigen::Vector3d &other) {
     return Angles::angle_diff_rad(az, quat_.yaw());
 }
 
+Eigen::Matrix4d State::tf_matrix(bool enable_translate)
```

Comment:
It seems like this is doing something very similar to Eigen's [toRotationMatrix](http://eigen.tuxfamily.org/dox/classEigen_1_1RotationBase.html#a7f2232646af917ad28bc6acc228e0f75) and [Transform](http://eigen.tuxfamily.org/dox/classEigen_1_1Transform.html). Can we just use those functions?